src/components/FeedPagination.vue

```diff
@@ -41,10 +41,16 @@ export default {
     address: { type: String, default: null },
     search: { type: String, default: null },
     blacklist: Boolean,
+    feedTips: Boolean,
+    feedPosts: Boolean,
```

Comment:
```suggestion
    tips: Boolean,
    posts: Boolean,
```
this component is already named FeedPagination

Reply:
there's already this.tips from the store. Can we keep it this way or rename it something else